index.js

```diff
@@ -1,15 +1,37 @@
 /* eslint-env node */
 'use strict';
 
+const Path = require('path');
+const Funnel = require('broccoli-funnel');
+const MergeTrees = require('broccoli-merge-trees');
+
+function isNotFastBoot() {
+  return typeof FastBoot === 'undefined';
```

Comment:
just FYI that this is now the preferred method for detecting Fastboot. As of the latest RCs the env var is gone.